Makefile

```diff
@@ -146,9 +146,14 @@ docker-build: ## to build the docker image
 .PHONY: docker-push
 docker-push: ## to push the docker image
 	@$(INFO) Pushing to registry...
-	$(AT)$(DOCKER) tag ${APP_NAME}:${APP_VERSION} $(DOCKER_REGISTRY)/${DOCKER_REGISTRY_REPO}/${APP_NAME}:${APP_VERSION} || ${FAIL}
-	$(AT)$(DOCKER) push $(DOCKER_REGISTRY)/${DOCKER_REGISTRY_REPO}/${APP_NAME}:${APP_VERSION} || ${FAIL}
-	@$(OK) Pushing to registry $(DOCKER_REGISTRY)/${DOCKER_REGISTRY_REPO}/${APP_NAME}:${APP_VERSION}
+	$(AT)$(DOCKER) tag ${APP_NAME}:${APP_VERSION} $(DOCKER_REGISTRY)/${DOCKER_REGISTRY_REPO}:${APP_VERSION} || ${FAIL}
+	$(AT)$(DOCKER) push $(DOCKER_REGISTRY)/${DOCKER_REGISTRY_REPO}:${APP_VERSION} || ${FAIL}
+# if we are on a tag, also push latest
+ifneq ($(shell echo $(APP_VERSION) | egrep '^v([0-9]+\.){0,2}(\*|[0-9]+)'),)
```

Comment:
What if we are at older tag. For instance assume that we have a 6.6.0 release yesterday and it was pushed as latest.

Today what if we decided to publish 6.5.1 patch release. Latest tag should still point to 6.6.0.

Reply:
That's a good point. I've assumed that release always iterates on the previous release. 
Is that a possibility?

Reply:
I guess it is , if we push a patch on a previous minor version. 
Will refactor the code

Reply:
Rebased and pushed changes

Reply:
:heart: Nice one!